.github/workflows/build-test-release.yml

```diff
@@ -121,7 +125,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        splunk-version: ["8.0","8.1","8.2"]
+        splunk-version: ["8.1","8.2"]
```

Comment:
My feeling it that should be 3 most recent versions

Reply:
Splunk 8.0 is not supported anymore: https://www.splunk.com/en_us/legal/splunk-software-support-policy.html